serverless/openvino/omz/intel/person-reidentification-retail-300/nuclio/reid.py

```diff
@@ -0,0 +1,76 @@
+# Copyright (C) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import math
+import numpy
+
+from scipy.optimize import linear_sum_assignment
+from scipy.spatial.distance import euclidean, cosine
+
+from model_loader import ModelLoader
+
+class ReID:
+    def __init__(self, model_xml, model_bin):
+        self.model = ModelLoader(model_xml, model_bin)
```

Comment:
Codacy found an issue: [Too many positional arguments for constructor call](https://app.codacy.com/manual/cvat/cvat/pullRequest?prid=5729179)